hack/configapi/cmd/delete.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfig/client"
+	"github.com/edgelesssys/constellation/v2/internal/staticupload"
+	"github.com/spf13/cobra"
+)
+
+// newDeleteCmd creates the delete command.
+func newDeleteCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "delete a specific version from the config api",
+		RunE:  runDelete,
+	}
+	cmd.PersistentFlags().StringP("version", "v", "", "Name of the version to delete (without .json suffix)")
+	must(enforceRequiredFlags(cmd, "version"))
+	return cmd
+}
+
+type deleteCmd struct {
+	attestationclient client.Client
```

Comment:
Instead of importing the `client.Client` interface here, define a new interface in this file that only has the relevant functions for `delete`.

Example:
```Go
type deleteClient interface {
    DeleteAzureSEVSNVersion(ctx context.Context, versionStr string) error
}

That way the implementation of delete is a lot more flexible, since we only depend on the client implementation to implement the function we actually use in the code, and writing unit test is a bit easier, since we can skip out on writing stub functions that are never called.